src/components/Card.tsx

```diff
@@ -0,0 +1,133 @@
+import { formatTime } from "./FormattTime";
+import "../css/Card.css";
+import { useRef, useEffect, useState } from "react";
+import { MODALS_ID } from "../modal/modalcomponents/ModalId";
+import AlertModal from "../modal/AlertModal";
+import AddLinkModal from "../modal/AddLinkModal";
+import kebob from "../image/kebab.png";
+import starImage from "../image/star.png";
+
+function formatDate(value: number) {
+  const date = new Date(value);
+  return `${date.getFullYear()}. ${date.getMonth() + 1}. ${date.getDate()}`;
+}
+interface Item {
+  url: string;
+  imageSource: string;
+  image_source: string;
+  title: string;
+  createdAt: number;
+  created_at: number;
+  description: string;
+}
+
+interface CardProps {
+  item: Item;
+  folderId: string;
+  userFolder: { data: { id: number; name: string; link: { count: number } }[] };
+  links: any;
+}
+export function Card({ item, folderId, userFolder, links }: CardProps) {
+  const [kebabPopUp, setKebabPopUp] = useState<boolean>(false);
+  const [currentModal, setCurrentModal] = useState<string | null>(null);
+  const [selectedFolderId, setSelectedFolderId] = useState<number | null>(
+    parseInt(folderId)
+  );
+
+  const kebabRef = useRef(null);
+  const closeModal = () => setCurrentModal(null);
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === "Escape") {
+      closeModal();
+    }
+  };
+  useEffect(() => {
+    function handleClickOutside(event: MouseEvent) {
+      if (
+        kebabRef.current &&
+        !(kebabRef.current as HTMLElement).contains(event.target as Node)
+      ) {
+        setKebabPopUp(false);
+      }
+    }
+    document.addEventListener("mousedown", handleClickOutside);
+  }, []);
+
+  const handlePopup = () => {
+    setKebabPopUp(!kebabPopUp);
+  };
+  return (
+    <>
+      {window.location.pathname === "/shared" ? (
```

Comment:
P2: path에 따라 컴포넌트가 다르게 렌더링되는데 각자 분리해주시면 가독성이 더 좋겠네요.

```suggestion
  return {window.location.pathname === "/shared" ? <SharedCard/> : <FolderCard/>}
```